Revert "[DTensor] Refactor strategy/rule registration into dedicated module (#168221)"

wconstab opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #170827
* #170359
* #167677
* __->__ #170615

This reverts commit cb3754f39fce0ce5162fc25f3c7cd0ab451ecd79.

Reverting this change as it affects the import path of a publicly
used API.

wconstab committed:
Revert "[DTensor] Refactor strategy/rule registration into dedicated module (#168221)"

This reverts commit cb3754f39fce0ce5162fc25f3c7cd0ab451ecd79.

Reverting this change as it affects the import path of a publicly
used API.

[ghstack-poisoned]

wconstab committed:
Update on "Revert "[DTensor] Refactor strategy/rule registration into dedicated module (#168221)""


This reverts commit cb3754f39fce0ce5162fc25f3c7cd0ab451ecd79.

Reverting this change as it affects the import path of a publicly
used API.

[ghstack-poisoned]

wconstab committed:
Update on "Revert "[DTensor] Refactor strategy/rule registration into dedicated module (#168221)""


This reverts commit cb3754f39fce0ce5162fc25f3c7cd0ab451ecd79.

Reverting this change as it affects the import path of a publicly
used API.

[ghstack-poisoned]

wconstab committed:
Update on "Revert "[DTensor] Refactor strategy/rule registration into dedicated module (#168221)""


This reverts commit cb3754f39fce0ce5162fc25f3c7cd0ab451ecd79.

Reverting this change as it affects the import path of a publicly
used API.

[ghstack-poisoned]

wconstab committed:
Update on "Revert "[DTensor] Refactor strategy/rule registration into dedicated module (#168221)""


This reverts commit cb3754f39fce0ce5162fc25f3c7cd0ab451ecd79.

Reverting this change as it affects the import path of a publicly
used API.

[ghstack-poisoned]

jeanschmidt commented:
@pytorchbot revert -m "Required to revert https://github.com/pytorch/pytorch/pull/170030" -c ghfirst

jeanschmidt commented:
@pytorchbot revert -m "Required to revert https://github.com/pytorch/pytorch/pull/170030" -c ghfirst

wconstab committed:
Update on "Revert "[DTensor] Refactor strategy/rule registration into dedicated module (#168221)""


This reverts commit cb3754f39fce0ce5162fc25f3c7cd0ab451ecd79.

Reverting this change as it affects the import path of a publicly
used API.

[ghstack-poisoned]
